zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1822,6 +1823,46 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /ad-hoc-activities/search:
+    post:
+      tags:
+        - Ad-hoc subprocess
+      operationId: findAdHocSubprocessActivities
+      summary: Query activatable activities within ad-hoc subprocesses
```

Comment:
```suggestion
      summary: Query activatable activities
```

🔧 On the docs side, we prefer for summaries to be relatively short, since they appear in the left nav, and wordier summaries can make it harder to find what they're looking for. The user can tell from the navigation hierarchy that they are within the Ad-hoc subprocesses, so my suggestion removes that bit of context. If there are other ways you can think of to shorten, that's great too. 

(Here are screenshots of the docs nav with the original, compared to my suggestion:)

<img width="270" alt="image" src="https://github.com/user-attachments/assets/5644dbf4-6e31-4279-85d5-ddf02a345df9" />

<img width="273" alt="image" src="https://github.com/user-attachments/assets/3bd48608-1021-4029-a866-0ba4a9022ed2" />